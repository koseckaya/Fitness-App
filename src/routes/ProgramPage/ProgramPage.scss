@import "../../styles/vars";

.program-page {
  box-sizing: border-box;
  display: grid;
  grid-template-columns: 320px minmax(0px, 1fr);
  column-gap: 40px;
  background-color: #fff;
  padding-top: 30px;
  &__main {
    h3 {
          font-size: 30px;
          font-weight: 700;
          margin: 0;
    }
  }
}
.side {
  &__image {
    position: relative;
  }
  &__img {
    object-fit: contain;
    width: 320px;
    border-radius: 15px;
    margin-bottom: 20px;
  }
  &__duration, &__minutes{
    position: absolute;
    left: 8px;
    bottom: 16px;
    text-transform: lowercase;
    background: #fff;
   border-radius: 15px;
   font-size: 13px;
    padding: 3px;
    color: $color-primary;
  }
  &__minutes {
    left: 70px;
  }
  &__type {
    font-size: 14px;
    margin-bottom: 6px;
  }
  &__equip, &__desc {
    font-size: 14px;
    margin-bottom: 6px;
  }
  &__title {
    color: $secondary;
    font-weight: 700;
    font-size: 12px;
    text-transform: uppercase;
    margin-bottom: 10px;
    letter-spacing: 0.06em;
    position: relative;
    margin-top: 30px;
        padding-top: 10px;

    &::before {
      content: "";
      height: 1px;
      width: 100%;
      background: $secondary;
      position: absolute;
      top: 0;
      left: 0;
      right: 0;
    }
  }
 &__start {
   padding: 20px;
    border-radius: 15px;
    display: flex;
    flex-direction: column;
    background-color: #ece9fc;
    margin: 20px 0;

 }

}
.start {
  &__title {
    font-weight: 600;
    font-size: 18px;
    margin-bottom: 10px;
  }
  &__btn {
    margin: 15px auto 0;
      color: $color-primary;
      background-color: transparent;
      border: 1px solid $color-primary;
      transition: 0.5s all ease;
      opacity: 0.5;
      &:hover {
      opacity: 1;
      pointer-events: all;
    }
   }
    &__active{
      background-color: $color-primary;
      color: rgb(255, 255, 255);
      opacity: 1;
    }
     &__error{
      background-color: $error;
      color: rgb(255, 255, 255);
      opacity: 1;
    }
}
.schedule {
  background-color: white;
    border: 1px solid rgba(239, 240, 244, 0.8);
    border-radius: 15px;
    padding: 40px 40px 30px;
    display: flex;
    flex-direction: column;
    gap: 20px;
    width: 100%;
  &__head {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
  }
  &__title {
    font-weight: 600;
    font-size: 22px;
    line-height: 130%;
  }
  &__week {
    height: 30px;
    width: fit-content;
    padding: 5px 12px;
    border: 1px solid $color-primary;
    border-radius: 54px;
    background-color: transparent;
    margin-left: 5px;
    cursor: pointer;
    &.active {
       background-color: rgb(124, 115, 197);
        border-color: $purple;
        color: #fff;
    }
  }
}

@include for-max-1443px() {
  .program-page {
    padding: 20px;
    display: flex;
    flex-direction: column;
    &__side {
      display: grid;
      grid-template-columns: 1fr 2fr 1fr;
      grid-template-rows: repeat(3, auto);
      grid-template-areas:
      "image title start"
      "type desc start"
      "equip desc start";
      column-gap: 10px;
      .side__image {
      grid-area: image;
      }
      
      .side__type {
      grid-area: type;
      }
      
      .side__equip {
      grid-area: equip;
      }
      
      .side__title {
      grid-area: title;
      }
      
      .side__desc {
      grid-area: desc;
      }
      
      .side__start {
      grid-area: start;
      }
    }
    &__main {
      .schedule {
        padding: 0;
        border: none;
      }
    }
  }
}