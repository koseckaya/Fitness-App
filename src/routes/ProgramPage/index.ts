export { default as ProgramPage } from './ProgramPage';
export type { Props as ProgramPageProps } from './ProgramPage';
