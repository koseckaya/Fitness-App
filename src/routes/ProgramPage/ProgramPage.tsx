import { FC, useState, useCallback } from 'react';
import './ProgramPage.scss';
import { useParams } from "react-router-dom";
import { programs, Programs } from '../../data';
import { Container } from '../../components/Container';

export type Props = {
    className?: string;

};



const ProgramPage: FC<Props> = ({ className }: Props) => {
    let { programId } = useParams();
    const program: Programs = programs.filter(prog => prog.path === '/' + programId)[0]

    const [week, setWeek] = useState(1);

    const countWeek = Math.ceil(program.days / 7)
    const filterWeeks =[]
    for (let i = 1; i <= countWeek; i++) {
        filterWeeks.push({value: `Week ${i}`, key: i})
    }
    filterWeeks.push({ value: `View All`, key: 0 })
    
    const handleWeekClick = useCallback(
        (e: any) => {
            const target = e.target
            console.log('target', target.dataset.week);
        setWeek(+target.dataset.week)
      },
      [setWeek],)
    

    return (
        <Container>
            <div className='program-page'>
                <div className='program-page__side'>
                    <div className='side__image'>
                        <img className='side__img' src={program.imageUrl2} alt={program.title} />
                        <span className='side__duration'>{program.days} days</span>
                        <span className='side__minutes'>{program.time[0]}-{program.time[1]} min/day</span>
                    </div>
                    <div className='side__type'>{program.type.join(', ')}</div>
                    <div className='side__equip'>{program.equip.join(', ')}</div>
                    <div className='side__title'>DETAILS</div>
                    <div className='side__desc'>{program.description}</div>
                </div>

                <div className='program-page__main'>
                    <h3>{program.title}</h3> 
                    <div className='schedule'>
                        <div className="schedule__head">
                        <div className="schedule__title">Full Schedule List View</div>
                        <div className="schedule__weeks">
                            
                                {filterWeeks.map(i => {
                                    const classActive = week === i.key ? "active" : '';
                                    return <span className={`schedule__week ${classActive}` } key={i.key}
                                           onClick={handleWeekClick}
                                           data-week={i.key}>{i.value}
                                       </span>  
                                   
                                })
                            }
                            
                            </div>
                        </div>
                        
                        
                    </div>
                </div>
            </div>
            
            
        </Container>
        


    )
    
}


export default ProgramPage;
