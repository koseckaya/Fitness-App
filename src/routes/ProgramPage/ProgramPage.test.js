import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import ProgramPage from './ProgramPage';

const mockedUsedNavigate = jest.fn();

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useParams: () => ({ programId: "grow-a-booty-challenge"}),
}));


it('renders correctly', () => {
    const tree = renderer
        .create(<Router><ProgramPage /></Router>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});