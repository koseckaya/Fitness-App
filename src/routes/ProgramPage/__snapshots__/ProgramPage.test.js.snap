// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container "
>
  <div
    className="program-page"
  >
    <div
      className="program-page__side"
    >
      <div
        className="side__image"
      >
        <img
          alt="Grow A Booty Challenge"
          className="side__img"
          src="https://static.chloeting.com/programs/6306f2a687caf1cb72ebad58/375058a0-86d2-11ed-8daf-11eb0dd6c117.jpeg"
        />
        <span
          className="side__duration"
        >
          28
           days
        </span>
        <span
          className="side__minutes"
        >
          20
          -
          50
           min/day
        </span>
      </div>
      <div
        className="side__type"
      >
        Resistance, Booty, Abs & Core
      </div>
      <div
        className="side__equip"
      >
        Fitness Mat, Dumbbells, Resistance Bands
      </div>
      <div
        className="side__title"
      >
        DETAILS
      </div>
      <div
        className="side__desc"
      >
        This 4-week resistance challenge will help you grow a booty with progressive overload! Dumbbells & resistance bands are recommended, a bench can be useful too. 
 Beginners can try this with bodyweight or temporary alternatives like filled water bottles
      </div>
      <div
        className="side__start"
      >
        <div
          className="start__title"
        >
          Start This Challenge
        </div>
        <div
          className="start__desc"
        >
          When you start a challenge you can see your progress in your personal account
        </div>
        <div
          className="button start__btn "
        >
          Should Sign in to Start
        </div>
      </div>
    </div>
    <div
      className="program-page__main"
    >
      <h3>
        Grow A Booty Challenge
      </h3>
      <div
        className="schedule"
      >
        <div
          className="schedule__head"
        >
          <div
            className="schedule__title"
          >
            Full Schedule List View
          </div>
          <div
            className="schedule__weeks"
          >
            <span
              className="schedule__week active"
              data-week={1}
              onClick={[Function]}
            >
              Week 1
            </span>
            <span
              className="schedule__week "
              data-week={2}
              onClick={[Function]}
            >
              Week 2
            </span>
            <span
              className="schedule__week "
              data-week={3}
              onClick={[Function]}
            >
              Week 3
            </span>
            <span
              className="schedule__week "
              data-week={4}
              onClick={[Function]}
            >
              Week 4
            </span>
            <span
              className="schedule__week "
              data-week={0}
              onClick={[Function]}
            >
              View All
            </span>
          </div>
        </div>
        <div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                1
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 1 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                2
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 2 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                3
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 3 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                4
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 4 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                5
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 5 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                6
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 6 as Complete
            </div>
          </div>
          <div
            className="day-program"
          >
            <div
              className="day__title"
            >
              <span
                className="day__number"
              >
                Day 
                7
              </span>
              <div
                className="day__desc"
              >
                <span
                  className="day__count"
                >
                   
                  3
                   Workouts | 
                </span>
                <span
                  className="day__duration"
                >
                   
                  27
                   Mins
                </span>
              </div>
            </div>
            <div
              className="program__videos"
            >
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7t1qhpDLSi0&ab_channel=ChloeTing"
                >
                  <img
                    alt="5 Min Warm Up"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/63ae100649cf64962cb594e2/f081bd90-8a98-11ed-97ac-4b900c40c009.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    5 Min Warm Up
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=7LFcL6aj73o&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Abs and Butt"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306edbb319d6a2844ffec11/cd4684b0-276e-11ed-b1cf-338eb74818fa.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Abs and Butt
                  </div>
                </a>
              </div>
              <div
                className="exercise-video-prev"
              >
                <a
                  href="https://www.youtube.com/watch?v=kI9mXjMQWzA&feature=youtu.be&ab_channel=ChloeTing"
                >
                  <img
                    alt="Lower Body Cooldown"
                    className="exercise-video-prev__img"
                    src="https://static.chloeting.com/videos/6306efc337990d0691f93a35/8693ea80-276e-11ed-94c7-572dee6a46f7.jpeg"
                  />
                  <div
                    className="video-prev "
                    onClick={[Function]}
                  />
                  <div>
                    Lower Body Cooldown
                  </div>
                </a>
              </div>
            </div>
            <div
              className="button btn-complete
                
                "
            >
              Mark Day 7 as Complete
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
